native/src/hooks/useLoadDisclaimer.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback } from 'react'
+
+import { createDisclaimerEndpoint, PageModel } from 'api-client'
+
+import { determineApiUrl } from '../utils/helpers'
+import useLoadCityContent, { CityContentReturn } from './useLoadCityContent'
+
+type UseLoadDisclaimerProps = {
+  cityCode: string
+  languageCode: string
+}
+
+const useLoadDisclaimer = ({
```

Comment:
We have a lot of duplication in this loading hooks, e.g. useLoadDisclaimers and useLoadOffers are almost the same. I think these hook can nicely be refactored with using template method pattern.

Reply:
Okay, will try. What is template method pattern, do you have an example?

Reply:
See: https://en.wikipedia.org/wiki/Template_method_pattern

Reply:
done